src/main/kotlin/no/nav/familie/oppdrag/repository/OppdragLagerRepositoryJdbc.kt

```diff
@@ -37,8 +37,10 @@ class OppdragLagerRepositoryJdbc(val jdbcTemplate: JdbcTemplate) : OppdragLagerR
         }
     }
 
-    override fun opprettOppdrag(oppdragLager: OppdragLager) {
-        val insertStatement = "INSERT INTO oppdrag_lager VALUES (?,?,?,?,?,?,?,?,?)"
+    override fun opprettOppdrag(oppdragLager: OppdragLager, versjon: Int) {
+        val insertStatement = "INSERT INTO oppdrag_lager " +
```

Comment:
Kan bli litt mer lesbar med NamedParameterJdbcTemplate

Reply:
Takk for tips! Skal sjekke det ut :)